### PR TITLE
EDSC-3951: Update GraphQL queries to use new `params` parameter objects

### PR DESCRIPTION
**Pull request by Bccorb**
# Overview

### What is the feature?

Updated some lingering graphQL requests to use the `params` attribute with supported types.

### What is the Solution?

Updated GraphQL queries and calls.

### What areas of the application does this impact?

Impacts Project view, main search view, and focused collection view.

# Testing

### Reproduction steps

- Performed outline regression tests in compared by Prod and Dev.

### Attachments

None.

# Checklist

- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## SIDEBAR - SEEKING FEEDBACK
- I had to remove hasGranuleCounts from the query as that wasn't a type of either param. I want to have someone take a look at that in detail and let me know, if there is a way to add that back or is it depreciated anyway?

### CODE REVIEW

**Review by dpesall — APPROVED** [review]

File: static/src/js/actions/exportSearch.js [hunk 1]
```diff
@@ -132,78 +132,8 @@ export const exportSearchAsStac = (format) => (dispatch, getState) => {
...
-        serviceType: $serviceType
-        sortKey: $sortKey
-        spatialKeyword: $spatialKeyword
-        tagKey: $tagKey
-        temporal: $temporal
-        twoDCoordinateSystemName: $twoDCoordinateSystemName
-        limit: $limit,
-        cursor: $cursor
-      ) {
+    query SearchCollections($params: CollectionsInput) {
```
Comment (eudoroolivares2016) [inline static/src/js/actions/exportSearch.js:135]: this function might be OBE I don't see any way to access it on the controller and its not called anywhere else

File: serverless/src/retrieveGranuleLinks/fetchCmrLinks.js [hunk 2]
```diff
@@ -8,58 +8,8 @@ import { getS3Urls } from '../../../sharedUtils/getS3Urls'
...
-      options: $options
-      orbitNumber: $orbitNumber
-      point: $point
-      polygon: $polygon
-      readableGranuleName: $readableGranuleName
-      sortKey: $sortKey
-      temporal: $temporal
-      twoDCoordinateSystem: $twoDCoordinateSystem
-    ) {
+  query GetGranuleLinks( $params: GranulesInput ) {
```
Comment (eudoroolivares2016) [inline serverless/src/retrieveGranuleLinks/fetchCmrLinks.js:11]: I am seeing an issue on this one where we can't get the granule downloads due to this causing a 500 error. To replicate: Find a collection add some granules to project use the Download option. Normally we get the granules links on the results page but, its erroring out

**Review by eudoroolivares2016 — CHANGES_REQUESTED** [review]

**Review by eudoroolivares2016 — COMMENTED** [review]
I added a commit that seems to fix the issue locally. We needed to wrap the request in a `params` object but, also there were issues calling the `(prepareGranuleAccessParams(granuleParams)` because the `pageSize` field is not defined for granules in graphql. See https://github.com/nasa/cmr-graphql/blob/main/src/types/granule.graphql

**Review by eudoroolivares2016 — CHANGES_REQUESTED** [review]
Blocking step as I need to update the sprint in bamboo after merges get done

**Pull request merged**